src/Slider.xml

```diff
@@ -9,33 +9,48 @@
         <property key="valueAttribute" required="false" type="attribute">
             <caption>Value attribute</caption>
             <category>Data source</category>
-            <description>The selected value on the slider</description>
+            <description>The selected value on the slider, if not provided 'static value' will be used</description>
```

Comment:
not static value, revert to previous description
<description>The selected value on the slider</description>